managed/models/service_helpers.go

```diff
@@ -123,6 +124,33 @@ func FindServices(q *reform.Querier, filters ServiceFilters) ([]*Service, error)
 	return services, nil
 }
 
+// FindActiveServiceTypes returns all active Service Types.
+func FindActiveServiceTypes(q *reform.Querier) ([]ServiceType, error) {
+	query := fmt.Sprintf(`SELECT DISTINCT service_type FROM %s`, ServiceTable.s.SQLName)
+	rows, err := q.Query(query)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	defer func() {
+		if err := rows.Close(); err != nil {
+			logrus.Debug(err)
+		}
+	}()
```

Comment:
Here we will overwrite error returned on line 145

Reply:
Ah, no, I'm wrong.

Reply:
BTW can we use different name for error variable here, to make it more obvious?

Reply:
Sure